arcsi/model/user.py

```diff
@@ -27,3 +28,20 @@ class User(db.Model, UserMixin):
 
     def __repr__(self):
         return "<Host {}>".format(self.name)
+
+class User2(db.Model, SQLAlchemyUserMixin):
+    """
+    A generic user model that might be used by an app powered by flask-praetorian.
+    Extends the SQLAlchemyUserMixin so that required methods and properties are already
+    well defined
+    """
+
+    __tablename__ = "users2"
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.Text, unique=True)
+    hashed_password = db.Column(db.Text)
+    roles = db.Column(db.Text)
+    is_active = db.Column(db.Boolean, default=True, server_default="true")
+
```

Comment:
Where is the api key column handled?

Reply:
In the same example I mentioned before I didn't see any field for it. Also I'm not sure about storing it in the db is a good idea, because it's going to be expired after 48 hours (or the time we set up in the config). I thought it is easier that if we need it we will just call the API (the proper, not the current hardcoded one) and it gives back the up-to-date token for us.